src/data/linux-quiz.ts

```diff
@@ -1330,6 +1330,54 @@ const linuxQuiz = [
     Distractor3: "stop [signal] PID",
     Explanation: "The kill command is used to stop a running process.",
     Link: "https://www.linuxfoundation.org/blog/blog/classic-sysadmin-how-to-kill-a-process-from-the-command-line"
+  },
+  {
+    Question:
+      "What is the purpose of the 'rsync' command in Linux, and how is it used?",
+    Answer:
+      "The 'rsync' command in Linux is used for data synchronization and file copying between directories or systems. It efficiently transfers and synchronizes files and directories while minimizing data transfer. It is often used for backup and mirroring purposes.",
+    Distractor1: "To restart a Linux system",
+    Distractor2: "To resize a partition",
+    Distractor3: "To install software packages",
+    Explanation:
+      "Rsync is a powerful tool for data synchronization, backup, and mirroring. It can be used locally or remotely to copy and synchronize files and directories with options to preserve permissions, timestamps, and more.",
+    Link: "https://liux.die.net/man/1/rsync"
+  },
+  {
+    Question:
+      "Explain the purpose of the 'cron' and 'at' commands in Linux. How do they differ?",
+    Answer:
+      "The 'cron' and 'at' commands in Linux are used for scheduling tasks. 'Cron' is a time-based job scheduler that allows users to automate recurring tasks at specified intervals. 'At' is used for one-time scheduling of tasks to run at a specific time in the future.",
+    Distractor1: "To create user accounts",
+    Distractor2: "To list installed packages",
+    Distractor3: "To check system resource usage",
+    Explanation:
+      "Cron is ideal for tasks like daily backups or system maintenance, while 'at' is suited for one-off tasks like scheduling a script to run at a specific time.",
+    Link: "https://linux.die.net/man/1/cron, https://linux.die.net/man/1/at"
```

Comment:
link leads to 404 not found

```suggestion
    Link: "https://www.freecodecamp.org/news/cron-jobs-in-linux/"
```